resource/utilities/rq2.cpp

```diff
@@ -222,14 +222,81 @@ static void usage (int code)
     exit (code);
 }
 
+double get_elapse_time (timeval &st, timeval &et)
```

Comment:
I think the "elapse_time" was a typo in the original `resource-query` that should be changed to `get_elapsed_time`. Regrettably, I think I perpetuated this typo in my API implementation...